terasoluna-gfw-functionaltest-web/src/main/resources/META-INF/spring/spring-security.xml

```diff
@@ -6,7 +6,8 @@
         http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
     ">
 
-  <sec:http pattern="/logging/**">
+  <sec:http pattern="/logging/**" use-authorization-manager="true" request-matcher="ant">
```

Comment:
The default value of use-authorization-manager is true, so there is no need to set it.

Reply:
Fixed according to review comments
fb66dc4